src/test/java/edu/harvard/iq/dataverse/api/UsersIT.java

```diff
@@ -206,15 +207,13 @@ public void testMergeAccounts(){
         String aliasInOwner = "groupFor" + dataverseAlias;
         String displayName = "Group for " + dataverseAlias;
         String user2identifier = "@" + usernameConsumed;
+        String target2identifier = "@" + targetname;
         Response createGroup = UtilIT.createGroup(dataverseAlias, aliasInOwner, displayName, superuserApiToken);
         createGroup.prettyPrint();
         createGroup.then().assertThat()
                 .statusCode(CREATED.getStatusCode());
 
-        String groupIdentifier = JsonPath.from(createGroup.asString()).getString("data.identifier");
-
-        List<String> roleAssigneesToAdd = new ArrayList<>();
-        roleAssigneesToAdd.add(user2identifier);
+        List<String> roleAssigneesToAdd = Arrays.asList(user2identifier, target2identifier);
```

Comment:
@stevenferey does it make sense to leave in the original test (that is, create another group and only add the "consumed" user) - just so we don't lose the original test?

Reply:
The original test have been remove because if the test is valid for merged accounts in the same group, I think it's difficult to not to be valid with accounts not in the same group. However, I don't think the reverse is true.

But I can keep the original test and add a specific test for accounts in the same group.

Reply:
No, it's OK - thanks. Just wanted to make sure it was considered.